src/client/client.tsx

```diff
@@ -42,15 +47,23 @@ export function fetchAllEvents(): Promise<Event[]> {
  * @param nuid The nuid of the member
  * @returns The Member with that nuid or undefined
  */
-export function fetchMember(nuid: string): Promise<Member | undefined> {
-  return new Promise((resolve, reject) => {
-    setTimeout(() => {
-      const member = (UserJSON as unknown as Member[]).find(
-        (m) => m.nuid === nuid
-      );
-      resolve(member);
-    });
-  });
+export async function fetchMember(
+  nuid: string
+): Promise<GenericResponse<Member>> {
+  const response = await axios.get(
+    `${process.env.REACT_APP_API_ENDPOINT}/api/member/getMember/?id=${nuid}`
```

Comment:
Just curious does this work when you locally tested it, throwing in the route from your PR I get blocked on access control

Reply:
yep, worked locally for me

Reply:
Did you use localhost? that worked for me,  I was getting blocked by cors when I swapped out the route with the route created from the vercel deploy on your backend pr

Reply:
oh, we can't actually test against https, it would all have to be done locally

Reply:
mfa and graduate use a proxy to deal with cors, but idk how that works with vercel functions

Reply:
seems like it can be done similar to what mfa does: https://mmazzarolo.com/blog/2022-02-05-creating-and-deploying-a-proxy-server-in-5-minutes/ , I haven't tested it, but there's a source from a quick google search